test/functional/column-chart.spec.js

```diff
@@ -139,11 +139,48 @@ fdescribe('column-chart.js', function(){
 
         xit('to display empty month columns even if date is missing', function(){
             var txt = document.querySelectorAll('.width600 svg')[2].querySelectorAll('g.x.axis .primary text');
-//todo: Months card is later!
+            //todo: Months card is later!
             expect(txt[0].textContent).toBe('Mar');
             expect(txt[1].textContent).toBe('Apr');
             expect(txt[2].textContent).toBe('May');
             expect(txt[3].textContent).toBe('Jun');
         });
+
+        it('are less than the y axis maximum', function(){
+            var cols = document.querySelectorAll('.width600 svg')[7].querySelectorAll('.plot rect');
+            var ticks = document.querySelectorAll('.width600 svg')[7].querySelectorAll('.y.axis .tick text');
+            var lastTick = ticks[ticks.length - 1];
+
+            for(var i = 0; i < cols.length; i++){
+                expect(cols[i].__data__.value).toBeLessThan(Number(lastTick.textContent));
+            }
+        });
+    });
+
+    xdescribe('number of columns ', function(){
+        it('in day chart is correct', function(){ //this should fail because the number of columns showing in a day is 1 and should be 4
+            var cols = document.querySelectorAll('.width600 svg')[6].querySelectorAll('rect');
+
+            expect(cols.length).toBe(4);
+        });
+
+        it('in hour chart is correct', function(){ //this should fail because the number of columns showing in a hour is 1 and should be 4
+            var cols = document.querySelectorAll('.width600 svg')[7].querySelectorAll('rect');
+
+            expect(cols.length).toBe(4);
+        });
+    });
+
+    describe('stacked column chart ', function(){ //still working on this one, had to leave a bit early
+        it('has the same number of stacks in each grouping', function(){
+            var g = document.querySelectorAll('.width600 svg')[9].querySelectorAll('.plot g.series');
+            var i = g.length;
+            var firstGroup = g[0].querySelectorAll('rect');
+
+            while(i--){
+                var thisGroup = g[i].querySelectorAll('rect');
+                expect(thisGroup.length).toBe(firstGroup.length);
+            }
+        });
```

Comment:
Quantity of columns is in there but it is currently x'd out so not as to throw an error on the "day" and "hour" charts